src/applications/coronavirus-screener/components/FormQuestion.jsx

```diff
@@ -7,6 +9,12 @@ export default function FormQuestion({
   scrollNext,
 }) {
   function handleChange(event) {
+    if (_.isEmpty(formState)) {
+      recordEvent({
+        event: 'covid-screening-tool-button-click',
```

Comment:
it seems like this event name should be like `covid-screening-tool-start` for this version because the event is not recording every button click? i know that differs from https://github.com/department-of-veterans-affairs/va.gov-team/issues/9255#issuecomment-631615453

Reply:
just caught up on the discussion about this here: https://github.com/department-of-veterans-affairs/va.gov-team/issues/9255

Reply:
@mchelen-gov I agree and have requested that event name.